packages/cli/.mocharc.js

```diff
@@ -0,0 +1,53 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: @loopback/cli
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+// A workaround to use `--require ./test/snapshot-matcher.js` so that root
+// hooks are executed by mocha parallel testing for each job
+
+const debug = require('debug')('loopback:cli:test');
+
+/**
+ * Build mocha config for `@loopback/cli`
+ */
+function buildConfig() {
+  // Use the default config from `@loopback/build`
+  const mochaConfig = require('@loopback/build/config/.mocharc.json');
+  debug('Default mocha config:', mochaConfig);
+  // Resolve `./test/snapshot-matcher.js` to get the absolute path
+  const mochaHooksFile = require.resolve('./test/snapshot-matcher.js');
+  debug('Root hooks for --require %s', mochaHooksFile);
+  const config = {...mochaConfig, timeout: 5000};
+
+  // Allow env var `MOCHA_JOBS` to override parallel testing parameters
+  const jobs = +process.env.MOCHA_JOBS;
```

Comment:
I feel we don't need to support `process.env.MOCHA_JOBS` as it's already possible to change the number of jobs (worker processes) by using `-j` command line argument and disable parallel execution by setting the number of jobs to 1.

```
$ npm run mocha -- -j 1
```